engine/dlib/src/dlib/buffer.h

```diff
@@ -74,6 +83,10 @@ Result GetStreamOffset(HBuffer buffer, uint32_t index, uint32_t* offset);
 
 Result CalcStructSize(uint32_t num_streams, const StreamDeclaration* streams, uint32_t* size, uint32_t* offsets);
 
+Result UpdateBounds(HBuffer hbuffer, dmhash_t stream_name);
+
+Result GetBounds(HBuffer hbuffer, Bounds3D** bounds);
```

Comment:
Replace with a SetBounds(hbuffer, min, max);

Reply:
Ok for min/max as points. But how will `SetBounds()` know which stream to process ?

Reply:
The interface Get/SetBounds has been reworked as suggested. I'll resolve this.